Pod/Classes/Math/MathHelpers.swift

```diff
@@ -30,31 +30,24 @@
 
 import Foundation
 
-public struct MathHelpers {
-
-    /**
-     Clamp a value to a ClosedInterval
-
-     - parameter value: the value to be clamped
-     - parameter to:    a ClosedInterval whose start and end specify the clamp's minimum and maximum
-
-     - returns: the clamped value
-     */
-    static func clamp<T: Comparable>(_ value: T, to: ClosedRange<T>) -> T {
-        return clamp(value, min: to.lowerBound, max: to.upperBound)
+public extension SignedNumber {
```

Comment:
#req why limit this to `SignedNumber` at a glance it looks like this can be written as an extension on `comparable`, or this should be extended to allow clamping of unsigned integers